server/apps/immich/src/api-v1/asset/dto/get-asset-thumbnail.dto.ts

```diff
@@ -16,4 +16,11 @@ export class GetAssetThumbnailDto {
     enumName: 'ThumbnailFormat',
   })
   format: GetAssetThumbnailFormatEnum = GetAssetThumbnailFormatEnum.WEBP;
+
+  @IsOptional()
+  @ApiProperty({
+    type: String,
+    required: false,
+  })
```

Comment:
API properties are automatically added using the swagger plugin, so this should not be necessary.

Reply:
It does if the DTO is a post body. I have problem with the DTO is the params or query, therefore you have to explicitly put it here.